src/gui/editors/AutomationEditor.cpp

```diff
@@ -654,6 +671,52 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 				}
 			break;
 			}
+			case EDIT_TANGENTS:
+			{
+				if (m_clip->progressionType() == AutomationClip::CubicHermiteProgression)
```

Comment:
I think it should be better to have a function in `AutomationClip` which returns if the progression type supports manual tangent edit, instead of hard-coding `AutomationClip::CubicHermiteProgression`.